homeassistant/components/smhi/weather.py

```diff
@@ -135,7 +136,7 @@ def __init__(
         """Initialize the SMHI weather entity."""
 
         self._attr_name = name
-        self._attr_unique_id = f"{latitude}, {longitude}"
+        self._attr_unique_id = f"smhi-{latitude}-{longitude}"
```

Comment:
Same question here. The entity registry is aware of both the integration and the platform domain.